src/appMain/CMakeLists.txt

```diff
@@ -30,6 +30,15 @@
 cmake_minimum_required(VERSION 2.8.11)
 
 if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+add_custom_target(copy_sqlite_dll ALL
```

Comment:
@OHerasym, add indent here and remove empty line #38.

Also seems this will work for both Qt and Windows. Ensure that this fix apply only for win native. Qt doesn't require this dll.